instrumentation/kafka-clients/kafka-clients-common/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/kafka/KafkaTracingWrapperUtil.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.instrumentation.kafka;
+
+public final class KafkaTracingWrapperUtil {
```

Comment:
I wonder if we could rename it to sth like `KafkaClientsConsumerProcessTracing` - since it only affects the CONSUMER `operation = "process"` part of kafka-clients instrumentation